fs/config/config.go

```diff
@@ -126,3 +128,34 @@ type BackgroundFetchConfig struct {
 	// fetcher emits metrics
 	EmitMetricPeriodSec int64 `toml:"emit_metric_period_sec"`
 }
+
+type RetryableHTTPClientConfig struct {
```

Comment:
Since they're exported structs, can we add a brief comment to reduce some fmt warnings?